tpot/base.py

```diff
@@ -929,6 +932,15 @@ def _save_periodic_pipeline(self):
         except Exception as e:
             self._update_pbar(pbar_num=0, pbar_msg='Failed saving periodic pipeline, exception:\n{}'.format(str(e)[:250]))
 
+    def _create_periodic_checkpoint_folder(self):
+        try:
+            os.makedirs(self.periodic_checkpoint_folder)
+        except OSError as e:
+            if e.errno == errno.EEXIST and os.path.isdir(self.periodic_checkpoint_folder):
+                pass
```

Comment:
We need add a warning message to let user know if the folder exists or tpot make a new folder.

Reply:
I added a line to output a message via the progress bar when the folder is created. I didn't replace the `pass` with a message because I'd expect that to be the default path - the user will probably create a folder so I'm not sure they need a message then, and if they don't create one we probably only want to show the message about the automatic folder creation once. Is that ok?

Reply:
oic, maybe just a friendly message when a new folder is made would be fine.